drivers/i2c/i2c_dw.c

```diff
@@ -938,6 +941,33 @@ static int i2c_dw_initialize(const struct device *dev)
 	_CONCAT(I2C_DW_IRQ_FLAGS_SENSE, DT_INST_IRQ_HAS_CELL(n, sense))(n)
 
 /* not PCI(e) */
+#if DT_ANY_INST_PROP_STATUS_OKAY(aux_reg)
+#define I2C_DW_IRQ_CONFIG_PCIE0(n)                                            \
```

Comment:
Instead of overloading this 'CONFIG_PCIE0', wouldn't it be better to put the #ifdef where the I2C_DW_IRQ_CONFIG_PCIE0 macro is used? That avoids this confusing PCIe reference.